db/migrations/script.py.mako

```diff
@@ -0,0 +1,24 @@
+"""${message}
```

Comment:
🤔 Could you explain the usage of this file please? :)

Reply:
Again auto-generated by Flask-Migrate so part of migrations management